rollup.config.js

```diff
@@ -20,8 +20,7 @@ const processEnv = conf => ({
 });
 
 const vuePlugin = process.env.VUE_VERSION === 'vue3' ? vueV3 : vueV2;
-const outputDir =
-  process.env.VUE_VERSION === 'vue3' ? 'dist/vue3' : 'dist/vue2';
+const outputDir = process.env.VUE_VERSION === 'vue3' ? 'vue3' : 'dist/vue2';
```

Comment:
should it not also be /vue2 then? instead of dist/vue2?

Reply:
yeah let's go with that